### PR TITLE
feat(TDL-28,25): added style to appear new task in fancy way, change icon to edit to icon from google icons

### PR DESCRIPTION
**Pull request by macopul**

### CODE REVIEW

**Comment by macopul** [comment]
Completed task 25 and 28 on this branch.

File: src/ToDoItem/ToDoItem.js [hunk 1]
```diff
@@ -9,20 +9,43 @@ export const ToDoItem = (message) => {
   checkbox.type = "checkbox";
   checkbox.classList.add("to-do-element-checkbox");
 
+  const checkToDoElement = (e) => {
+    e.target.checked
+      ? (toDoMessage.style.textDecoration = "line-through")(
```
Comment (woopul) [inline src/ToDoItem/ToDoItem.js:14]: toDoMessage.style.textDecoration = e.target.checked ? "line-through" :  "none"

Comment (woopul) [reply src/ToDoItem/ToDoItem.js:14]: if(e.target.checked) {

File: src/ToDoItem/ToDoItem.js [hunk 2]
```diff
@@ -9,20 +9,43 @@ export const ToDoItem = (message) => {
...
   const toDoMessage = document.createElement("p");
-  toDoMessage.innerText = "test message";
+  toDoMessage.classList.add("task-message");
+  toDoMessage.innerText = message;
+
+  //TODO: add button to edit TodoElement
+  const editButton = document.createElement("div");
+  editButton.classList.add("edit-button");
+
+  // const editToDoButton = (e) => {toDoMessage.contentEditable = e.target.checked ? false : true}
```
Comment (woopul) [inline src/ToDoItem/ToDoItem.js:30]: please remove comment here

File: src/ToDoItem/ToDoItem.js [hunk 3]
```diff
@@ -9,20 +9,43 @@ export const ToDoItem = (message) => {
...
+
+  const editToDoButton = () => {
+    if (checkbox.checked == false) {
```
Comment (woopul) [inline src/ToDoItem/ToDoItem.js:33]: if(checkbox.checked) { 
 return;
  }
  toDoMessage.contentEditable = true;
  toDoMessage.focus();

File: src/ToDoItem/ToDoItem.js [hunk 4]
```diff
@@ -9,20 +9,43 @@ export const ToDoItem = (message) => {
...
+      toDoMessage.contentEditable = true;
+      toDoMessage.focus();
+    } else toDoMessage.contentEditable = false;
```
Comment (woopul) [inline src/ToDoItem/ToDoItem.js:36]: if you want to leave it as it is, add curly bracets {} to else

**Review by woopul — CHANGES_REQUESTED** [review]

**Review by woopul — APPROVED** [review]

File: src/ToDoItem/ToDoItem.js (hunk 3 above)
Comment (macopul) [reply src/ToDoItem/ToDoItem.js]: done in this way

File: src/ToDoItem/ToDoItem.js (hunk 1 above)
Comment (macopul) [reply src/ToDoItem/ToDoItem.js:14]: task on another branch #19

**Pull request merged**